src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext, useState } from 'react';
```

Comment:
This component should be under `src/Me/Routes/Home/` (`Home` folder doesn't exist, please create it)